### PR TITLE
Make Jenkins CI run against Betanet 5

### PR DESCRIPTION
**Pull request by Nazgolze**
### What was the problem?
This PR changes the Jenkins job to run against betanet5 

### How was it solved?

Modified Jenkinsfile.deployment, added relevant betanet5 docker compose and env files

### How was it tested?

Tested on Jenkins

### Notes
Also adds workaround for https://github.com/docker/for-linux/issues/264

### CODE REVIEW

File: docker/lisk_service/docker-compose.core.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ services:
     env_file:
       - ./lisk_service/env/common.env
       - ./lisk_service/env/core.env
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```
Comment (fchavant) [inline docker/lisk_service/docker-compose.core.yml:18]: Where does `host-gateway` come from?

Comment (Nazgolze) [reply docker/lisk_service/docker-compose.core.yml:18]: It's more magic :grin:

Comment (Nazgolze) [reply docker/lisk_service/docker-compose.core.yml:18]: I think it's pretty new too. Might be the very latest version of Docker even

Comment (fchavant) [reply docker/lisk_service/docker-compose.core.yml:18]: And this one only works on linux? Or why do we not use it directly in `LISK_CORE_*`?

File: Jenkinsfile.deployment [hunk 2]
```diff
@@ -39,17 +39,17 @@ pipeline {
 			when { branch 'development' }
 			environment {
 				ENABLE_HTTP_API='http-version1,http-version1-compat,http-status,http-test'
-				ENABLE_WS_API='rpc,rpc-v1,rpc-test,blockchain'
+				ENABLE_WS_API='rpc,rpc-v1,rpc-test,blockchain,rpc-v2'
```
Comment (MichalTuleja) [inline Jenkinsfile.deployment:42]: `ENABLE_WS_API='http-status,http-test,blockchain,http-version2,rpc-v2'`

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: docker/lisk_service/docker-compose.core.yml (hunk 1 above)
Comment (Nazgolze) [reply docker/lisk_service/docker-compose.core.yml:18]: This looks like the original PR https://github.com/moby/moby/pull/40007 
It's weird, but it looks like `host-gateway` can't just be used directly

**Review by fchavant — APPROVED** [review]

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**